### PR TITLE
Clean up example + update .eslintrc

### PR DESCRIPTION
**Pull request by erikshestopal**
- Properly use global style guide in `.eslintrc.json`
- Ran `eslint` on `examples/index.js` + removed `React.createClass`. This should help get rid of the some of the warnings in `commercial` when running `jest`.

### CODE REVIEW

**Comment by erikshestopal** [comment]
@berfarah can you take a look at this please?

File: examples/index.jsx [hunk 1]
```diff
@@ -28,21 +27,21 @@ const LANGUAGES_BUNDLE = {
 
 i18n.loadSync({
   'en-US': {
-    'langauges': LANGUAGES_BUNDLE,
-    'common': {
-      'header': '{ project } examples',
-      'helloWorld': 'Hello, {name}!',
-      'clicked': 'Click {count}',
-      'myLabel': "My Label",
+    langauges: LANGUAGES_BUNDLE,
```
Comment (berfarah) [inline examples/index.jsx]: I think languages is misspelled too

File: examples/index.jsx [hunk 2]
```diff
@@ -63,96 +62,99 @@ let CONSTANT_EXAMPLE;
...
 // Create Example Component
-const Examples = React.createClass({
-  getInitialState: function() {
-    return {
-      count: 0,
-    }
-  },
-  clickHandler: function () {
+class Example extends Component {
+  state = {
```
Comment (berfarah) [inline examples/index.jsx:68]: This doesn't have to happen in the constructor anymore?

**Review by berfarah — APPROVED** [review]
Seems legit. One non-blocking comment.

Also, I think it'd be totally fine to just allow `console.log` for this project, but I'm also fine with how you handled it

File: examples/index.jsx (hunk 2 above)
Comment (erikshestopal) [reply examples/index.jsx:68]: [Nope!](https://babeljs.io/blog/2015/06/07/react-on-es6-plus#property-initializers)

Comment (berfarah) [reply examples/index.jsx:68]: Neato!

**Pull request merged**